go-controller/pkg/config/config_test.go

```diff
@@ -749,6 +753,7 @@ var _ = Describe("Config Operations", func() {
 			"-metrics-enable-config-duration=true",
 			"-egressip-reachability-total-timeout=5",
 			"-egressip-node-healthcheck-port=4321",
+			"-enable-multi-network=true",
```

Comment:
I've pulled your PR, and changed this line to feature only `"-enable-multi-network"` and the test still passes. 

While the test is valuable, this didn't address my particular issue. 

Thanks for the help never the less, I'll keep digging.

Reply:
I am running into the same issue, I am trying to add a new flag in one of my PRs and its just not taking effect into the Config Params which is confusing..